docs/modules/ROOT/pages/configuration/observability.adoc

```diff
@@ -0,0 +1,43 @@
+= Observability on KubeArchive
+
+KubeArchive is instrumented to emit observability data: metrics,
```

Comment:
Passive tense:

```suggestion
OpenTelemetry instruments KubeArchive to emit observability data: metrics,
```

Reply:
Thanks for keeping an eye on this. Fixed.